src/vs/workbench/contrib/chat/electron-sandbox/actions/voiceChatActions.ts

```diff
@@ -1328,6 +1352,11 @@ export class InstallSpeechProviderForVoiceChatAction extends BaseInstallSpeechPr
 				when: HasSpeechProvider.negate(),
 				group: 'navigation',
 				order: 3
+			}, {
+				id: TerminalChatExecute,
```

Comment:
i'm seeing that when this is present, the duplicate mics show up. it seems `HasSpeechProvider` is always `false` for the terminal

Reply:
unclear if that's just what i'm seeing because the ext host is crashing in oss

Reply:
I hadn't installed the speech extension. Apparently in OSS, the notification to install doesn't show up, so this is expected.